src/test/unit/algorithms/SDRClassifierTest.cpp

```diff
@@ -150,30 +150,31 @@ TEST(ClassifierTest, MultipleCategories) {
 
 
 TEST(SDRClassifierTest, SaveLoad) {
-  ASSERT_EQ(1, 2); // TODO!
-  // vector<UInt> steps{ 1u };
-  // Predictor c1(steps, 0.1f, 0.1f);
-  // Predictor c2(steps, 0.1f, 0.1f);
-
-  // // Create a vector of input bit indices
-  // vector<UInt> input1{ 1u, 5u, 9u };
-  // vector<UInt> bucketIdxList1{4u};
-  // vector<Real64> actValueList1{34.7f};
-  // ClassifierResult result;
-  // c1.compute(0u, input1, bucketIdxList1, actValueList1, false, true, true, result);
-
-  // {
-  //   stringstream ss;
-  //   EXPECT_NO_THROW(c1.save(ss));
-  //   EXPECT_NO_THROW(c2.load(ss));
-  // }
-  // ASSERT_EQ(c1, c2);
-
-  // ClassifierResult result1, result2;
-  // c1.compute(1u, input1, bucketIdxList1, actValueList1, false, true, true, result1);
-  // c2.compute(1u, input1, bucketIdxList1, actValueList1, false, true, true, result2);
-
-  // ASSERT_EQ(result1[1u], result2[1]);
+  vector<UInt> steps{ 1u };
+  Predictor c1(steps, 0.1f);
```

Comment:
TODO also test for Classifier

Reply:
The `Predictor` contains a `Classifier` so this should test both at once.